### PR TITLE
Fix layering of dependencies in riot-web and e2e tests

### PR DESCRIPTION
**Pull request by turt2live**
~~**Requires https://github.com/vector-im/riot-web/pull/11953**~~

### CODE REVIEW

**Comment by turt2live** [comment]
riot-web tests are failing due to https://github.com/vector-im/riot-web/pull/11953 being missing. Pass locally, which is much closer to this setup than before.

This PR and/or https://github.com/vector-im/riot-web/pull/11956 fix the tests.

**Comment by jryans** [comment]
As Dave has been involved in the build system work and reviewed the PR this depends on, I'll adjust the reviewers to him.

**Comment by turt2live** [comment]
@jryans stuff like this should be reviewable by anyone, and should imo. These kinds of PRs are great to help break down the tribal knowledge barrier me and Dave are holding.

**Comment by jryans** [comment]
Okay, fair enough. I was mostly just following usual protocol since the other one has been reviewed. I agree it would be good to have more people aware of what's going on in the build system. 😄

File: scripts/fetchdep.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ clone() {
     if [ -n "$branch" ]
     then
         echo "Trying to use $org/$repo#$branch"
-        git clone git://github.com/$org/$repo.git $repo --branch "$branch" && exit 0
+        git clone git://github.com/$org/$repo.git $repo --branch "$branch" --depth=1 && exit 0
```
Comment (jryans) [inline scripts/fetchdep.sh]: Ultra nit, just to parallel how the other option is specified... 😅 

```suggestion
        git clone git://github.com/$org/$repo.git $repo --branch "$branch" --depth 1 && exit 0
```

**Comment by turt2live** [comment]
@jryans assuming I did everything right, the CI should be completely green now.

**Comment by turt2live** [comment]
it does! (for once)

File: .buildkite/pipeline.yaml [hunk 2]
```diff
@@ -76,34 +76,24 @@ steps:
...
 
   - label: "🔧 Riot Tests"
     agents:
       # We use a medium sized instance instead of the normal small ones because
       # webpack loves to gorge itself on resources.
       queue: "medium"
     command:
-      # Install chrome
-      - "echo '--- Installing Chrome'"
-      - "wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -"
```
Comment (bwindels) [inline .buildkite/pipeline.yaml]: Oh, I always though the karma tests relied still on this (e2e tests have puppeteer install it) :+1:

**Pull request merged**